app/views/articles/index.html.erb

```diff
@@ -11,11 +11,9 @@
      <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet">
   </head>
             <%= render 'shared/navbar-blog.html.erb' %>
```

Comment:
I suggest intenting same hierarchy components

Reply:
Hey Santi - Is this better now - can you review and then we merge?